.github/workflows/pfx_to_file.yml

```diff
@@ -1,22 +1,42 @@
 # Workflow to test secrets, first version empty in order to be able to run it from branch
-
 name: PFX To File
```

Comment:
What is the meaning of PFX?

Reply:
Stands for Personal File exchange format is used to exchange public and private objects in a single file.